### DISCUSSION TITLE
github.com/github/linguist: create zip: samples/Ren'Py/example.rpy: malformed file path "samples/Ren'Py/example.rpy": invalid char '\''

### DISCUSSION THREAD

**Discussion by ronniee007** [github-linguist/linguist#6492] [Classification]
This happens while doing go mod tidy command, takes in Ren'py as a malformed argument under file path is there any resolution for this to remove the char?

**Comment by lildude** [comment]
Can you provide more details of how you are using linguist to produce this error.

It sounds like you're trying to use the whole of linguist as a Go module in another app. This probably isn't what you want as the only Go code is the [grammar compiler](https://github.com/github-linguist/linguist/tree/master/tools/grammars) which isn't intended to be used by anything else. Linguist itself is written in Ruby. If you want a Go equivalent, take a look at [go-enry](https://github.com/go-enry/go-enry) which feeds off Linguist and is what GitHub used for its new Search.
